pkgs/games/koules/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, lib, fetchurl, makeDesktopItem, imake, gccmakedep, libX11, libXext }:
+
+let
+  desktopItem = makeDesktopItem {
+    desktopName = "Koules";
+    name = "koules";
+    exec = "xkoules";
+    icon = "koules";
+    comment = "Push your enemies away, but stay away from obstacles";
+    categories = "Game;ArcadeGame";
+  };
+  debian = fetchurl {
+    url = "mirror://debian/pool/main/k/koules/koules_1.4-27.debian.tar.xz";
+    sha256 = "0dac5957rikr2l9m5fsvmqqkf590rlhwxmdb5ghnm173wrk0bf00";
+  };
+in
+
+stdenv.mkDerivation rec {
+  name = "koules";
+  version = "1.4";
+
+  src = fetchurl {
```

Comment:
You should be able to use `fetchTarball` here, which removes the need to extract the tar file manually in `prePatch`.

Reply:
Good idea! Thanks, will do that!